### ISSUE TITLE
[Firefox] Addon icon displays red square with an "X" in it even when not in private mode

### ISSUE THREAD

**Issue by gfhesdf** [mrdokenny/Cookie-AutoDelete#123]
The icon for the addon shows a red square with a white "X" in it and when clicking the icon nothing happens (see attached image).

As far as i have understood it is supposed to do this when in private browsing mode, however I am not using private mode and it still does this. I have however set my browser to not save any browsing history, and when I change this setting to remember history then the addon works as supposed.

Is there any way to have the addon usable when not saving browsing history?

![icon](https://user-images.githubusercontent.com/31437426/29815670-d0ab7f32-8cb1-11e7-9bc1-7717bb65b6d9.png)

- OS/version: Windows 10
- Browser/version: Firefox 55.0.3 64-bit
- Cookie AutoDelete version: 1.4.0

**Comment by mrdokenny** [comment]
Are you on permanent private browsing mode?

![capture](https://user-images.githubusercontent.com/17077613/29835454-e7ab53d4-8ca6-11e7-9fec-907cc17acd7e.PNG)

**Comment by gfhesdf** [comment]
It would seem that I am. Even though I have not checked that box, I found the following quote on the Mozilla support base: "Using Never remember history is equivalent to always being in Firefox Private Browsing mode."

Thank you for helping me clear that up, now I know why the addon won't work. However, I have to ask, is there any reason as to why the addon refuses to operate in Private browsing mode? As far as i know the browser still stores cookies during the entire session (atleast mine still does) meaning that Cookie-Autodeletes function to clear cookies after a tab has been closed should still be useful. Or am i missing something here?

In any case, thank you for a great addon!
